.gitignore

```diff
@@ -41,3 +41,4 @@ test/client-old/spec/mocks/translations.js
 .elasticbeanstalk/*
 !.elasticbeanstalk/*.cfg.yml
 !.elasticbeanstalk/*.global.yml
+/nbproject/private/
```

Comment:
@AndrzejDoyle  Could you please undo your change to the .gitignore file? We use it for only files that will appear in everybody's local installs, rather than for files specific to certain set-ups. There's some other options for ignoring your own files here: https://help.github.com/articles/ignoring-files/